x-pack/test/cloud_security_posture_api/routes/benchmarks.ts

```diff
@@ -30,18 +33,6 @@ export default function (providerContext: FtrProviderContext) {
   const supertest = getService('supertest');
   const log = getService('log');
 
-  const getCspBenchmarkRules = async (benchmarkId: string): Promise<CspBenchmarkRule[]> => {
```

Comment:
Isn't removing this in favor of using hard coded mocks kinda defeats the purpose of this ftr test?

Reply:
This is causing the flakiness, this function can return undefined. Alternatively I can just make it so that when it returns undefined, we can just use a fallback value however if we do that wouldn't it be better to just use static mock instead

Reply:
i think we need to understand why it sometimes returns undefined and make sure the test environment have the required conditions for it not to return undefined

Reply:
hmmm true, that would be the best course of action
however debugging this is kinda tricky as it's somewhat hard to reproduce locally, 
are there any specific reason on why we are keen on not using Mocks specifically for this test ?